common/gc_topic/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/templates/topic/topic-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of effectiveness of Topic Title at appropiately describing the content of the page",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ""
```

Comment:
For the topic page, has you pre-assessed every HTML element or only the one contained in the `<main>` element?
Was there some sub-element that was excluded from the pre-assessment which would means that the assessment is limited to the integration of those component inside this template.

Reply:
Hi Pierre, I am not sure what this means. Which HTML elements  should I be pre-assessing for the topic page?